provisioning_client/devdoc/azure_provisioning_faq.md

```diff
@@ -0,0 +1,39 @@
+# Azure Iot Provisioning Client FAQ
+
+Here is where we shall try to answer question pertaining to the IoT Provisioning Client.  Please read the following documents for an overview of the DPS service.
+
+## x509 Provisioning
+
+- How does Provisining match the x509 certificates to an enrollment?
+
+  - The Provisioning Client add the x509 certificate (more on the certificate in a second) on the TLS stream.  The DPS service uses this certificate to look up the proper enrollment record.
```

Comment:
"add" should be "adds"

Please move the parenthetical (more on the cert...) to the end of the sentence so it breaks it up less.

"The DPS service" should be "The device provisioning service uses..."